README.md

```diff
@@ -277,6 +270,9 @@ android {
 
 Reference for more details https://github.com/ivpusic/react-native-image-crop-picker/issues/1406
 
+- If you use SDK version 33, add following to `app/src/main/AndroidManifest.xml`
+  - `<uses-permission android:name="android.permission.READ_MEDIA_IMAGES"/>`
```

Comment:
Do we need `android.permission.READ_EXTERNAL_STORAGE` ?

Reply:
From [the docs](https://developer.android.com/reference/android/Manifest.permission#READ_EXTERNAL_STORAGE):

> Starting in API level 33, this permission has no effect

So, no, we don't need it